pkg/minikube/storageclass/storageclass_test.go

```diff
@@ -0,0 +1,275 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storageclass
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	v1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+	storagev1 "k8s.io/client-go/kubernetes/typed/storage/v1"
+	testing_client "k8s.io/client-go/testing"
+)
+
+var orgGetStoragev1 func() (storagev1.StorageV1Interface, error)
+var orgMockStorageProvisioner, mockK8sConfig string
+
+func init() {
+	orgMockStorageProvisioner = "foo"
+	orgGetStoragev1 = getStoragev1
```

Comment:
Could this be refactored to not require code in `init()`?  Side effects here make tests difficult to follow.

Reply:
init function removed. Thanks!